main.go

```diff
@@ -226,6 +228,13 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&machineHealthCheckConcurrency, "machinehealthcheck-concurrency", 10,
 		"Number of machine health checks to process simultaneously")
 
+	fs.StringArrayVar(&machineSetPreflightChecks, "machineset-preflight-checks", []string{
+		string(clusterv1.MachineSetPreflightCheckAll)},
+		"List of MachineSet preflight checks that should be run. Per default all of them are enabled."+
+			"Set this flag to only enable a subset of them. The MachineSet preflight checks can be then also disabled"+
+			"on MachineSets via the 'machineset.cluster.x-k8s.io/skip-preflight-checks' annotation."+
+			"Valid values are: All, KubeadmVersionSkew, KubernetesVersionSkew, ControlPlaneIsStable, ControlPlaneVersionSkew")
```

Comment:
```suggestion
			"Valid values are: All or a comma separated list of KubeadmVersionSkew, KubernetesVersionSkew, ControlPlaneIsStable, ControlPlaneVersionSkew")
```

Reply:
Changed this also to StringSliceVar. Otherwise this would not be true